Source/Linq/Mock.cs

```diff
@@ -48,28 +48,54 @@
 
 namespace Moq
 {
-	public partial class Mock
-	{
-		/// <summary>
-		/// Creates an mock object of the indicated type.
-		/// </summary>
-		/// <typeparam name="T">The type of the mocked object.</typeparam>
-		/// <returns>The mocked object created.</returns>
-		public static T Of<T>() where T : class
-		{
-			return Mocks.CreateMockQuery<T>().First<T>();
-		}
+    public partial class Mock
+    {
+        /// <summary>
+        /// Creates an mock object of the indicated type.
+        /// </summary>
+        /// <typeparam name="T">The type of the mocked object.</typeparam>
+        /// <returns>The mocked object created.</returns>
+        public static T Of<T>() where T : class
+        {
+            return Mocks.CreateMockQuery<T>().First<T>();
+        }
```

Comment:
Would be nice if the github diff tool had an ignore whitespace option. Do we really need tabs here badly?